src/reporter.js

```diff
@@ -45,6 +46,10 @@ async function main (swapFuncParams: SwapFuncParams) {
     console.error('doShapeShift failed')
     return {}
   })
+  const rXai = await doSideshift(swapFuncParams).catch(e => {
```

Comment:
`doSideShift` or `doSideShift`? The latter would be in-line with `doShapeShift`.